### PR TITLE
Add memory-related tip

### PR DESCRIPTION
**Pull request by rprimet**
Suggest that starting the Minikube cluster with more memory may solve builder issues.

### CODE REVIEW

**Comment by consideRatio** [comment]
Excellent note! The default is currently 2GB according to [this minikube code](https://github.com/kubernetes/minikube/blob/20e6c5c579c7583c3ba3c79191c6ff3d71bd18e1/cmd/minikube/cmd/start.go#L95).

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -190,7 +190,7 @@ sudo apt install socat
    starts a local kubernetes cluster inside a virtual machine. This command
    assumes that you have already installed one of the VM drivers: virtualbox,
    xhyve or KVM2.
-
+
```
Comment (betatim) [inline CONTRIBUTING.md]:
```suggestion

```

Comment (betatim) [reply CONTRIBUTING.md]: If you merge this suggestion to remove whitespace we can merge this

**Comment by consideRatio** [comment]
Wiee thanks!

**Pull request merged**